src/lib/phdi-building-blocks/tests/test_geo.py

```diff
@@ -108,4 +108,4 @@ def test_geocode_patient_address(patched_geocoder):
     )
     patched_geocoder.return_value = geocoded_response
 
-    assert geocode_patient_address(raw_bundle, mock.Mock()) == standardized_bundle
+    assert geocode_patients_in_bundle(raw_bundle, mock.Mock()) == standardized_bundle
```

Comment:
Looks like we are missing a unit tests for `standardize_addresses_for_patient` and ` geocode_single_address`. I realize they are tested implicitly through the test for `geocode_patients_in_bundle`, however if we are splitting this functionality up into separate functions an end user might choose to use them outside of our functions that wrap them so I think we should write some explicit tests.

Reply:
I'm going to generate a series of tickets around ensuring our test coverage is as good as it can be for each module and function. While I wholeheartedly agree with what you're saying, Dan, this PR has been open for too long and is addressing too many things. This was an error on my part, and now I'm just trying to nip it in the bud, get it landed, and adjust for mistakes with some fast follow tickets, such as those for extending our tests to explicitly test newly created functions.